modules_basic/reply.js

```diff
@@ -0,0 +1,9 @@
+var h = require('../h')
+
+exports.gives = 'message_action'
+
+exports.create = function () {
```

Comment:
other creates take in api even though they don't use them. I think we should be consistent to help other people see the pattern